src/appmixer/utils/csv/AddRow/AddRow.js

```diff
@@ -23,34 +23,47 @@ module.exports = {
             parseNumbers,
             parseBooleans
         });
+        let lock;
+        let lockExtendInterval;
+        try {
+            lock = await context.lock(fileId, {
+                ttl: parseInt(context.config.lockTTL, 10) || 1000 * 60 * 1,
+                retryDelay: 1000
```

Comment:
1s delay, isn't that a lot? I would keep the default values and if anything, then make it configurable. No hardcoded numbers in the components!